starboard/build/toolchain/cobalt_toolchains.gni

```diff
@@ -23,6 +23,7 @@ template("cobalt_clang_toolchain") {
                              "executable_extension",
                              "tail_lib_dependencies",
                              "shlib_extension",
+                             "extra_cflags",
```

Comment:
this will go away if you address the gerrit comment

Reply:
done